src/FactorGraph01.jl

```diff
@@ -737,6 +737,8 @@ function doautoinit!(dfg::T,
         end
         pts,inferdim = predictbelief(dfg, vsym, useinitfct, logger=logger)
         setValKDE!(xi, pts, true, inferdim)
+        #TODO test
+        setVariablePosteriorEstimates!(xi)
```

Comment:
@dehann, I think adding extra data breaks compare test, but I haven't had time to dig into the details.

Reply:
these compare functions recently moved to DFG, so we should just tie up there too -- thanks!